nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -105,7 +105,7 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - The EC2 image module previously detected and automatically mounted ext3-formatted instance store devices and partitions in stage-1 (initramfs), storing `/tmp` on the first discovered device. This behaviour, which only catered to very specific use cases and could not be disabled, has been removed. Users relying on this should provide their own implementation, and probably use ext4 and perform the mount in stage-2.
 
-- `teleport` has been upgraded to major version 11. Please see upstream [upgrade instructions](https://goteleport.com/docs/setup/operations/upgrading/) and [release notes](https://goteleport.com/docs/changelog/#1100).
+- `teleport` has been upgraded from major version 10 to major version 12. Please see upstream [upgrade instructions](https://goteleport.com/docs/setup/operations/upgrading/) and release notes for versions [11](https://goteleport.com/docs/changelog/#1100) and [12](https://goteleport.com/docs/changelog/#1201). Note that Teleport does not officially support upgrades across more than one major version. It is recommended to first upgrade to an intermediate 11.x version using an overlay before upgrading to version 12.
```

Comment:
This was going to catch up with us eventually: Teleport has a more frequent release schedule (major version every ~3 months versus NixOS 6 months). 

I am not sure whether this warrants `stateVersion` and carrying several package versions (`teleport_10`, `teleport_11`, `teleport_12`). Teleport does not require manual steps in Postgres, but *in theory* something could break when upgrading 10 -> 12 (hence the recommendation to use 11 before upgrading to 12).

One more question: is it customary to link to a PR/commit in the release notes? It would be nice to link directly to https://github.com/NixOS/nixpkgs/pull/214425 here.

Reply:
exposing `teleport_11` would be nice.

Reply:
Linking to PR/commits in release notes is fine! Adds more context!

Reply:
> exposing `teleport_11` would be nice.

Will try to add multiple versions. Couple of questions:
* Do we add back `teleport_10` as well? Teleport supports latest + 2 previous major versions (10, 11, 12 currently).
* Do we update `nixosTests.teleport` to test each major version?